config/database.yml

```diff
@@ -1,11 +1,12 @@
 development: &default
   adapter: postgresql
   encoding: unicode
-  database: baseballsimulator_<%= Rails.env %>
+  database: baseballsimulator_development
   host: localhost
   pool: 5
   username: baseballsimulator
   password: baseball
 
-test: &test
+test:
   <<: *default
+  database: baseballsimulator_test
```

Comment:
Actually on second thought, `.travis.yml` doesn't care about development so this shouldn't be Travis's problem. Just be aware that Travis's database config file now no longer matches the main database config file. Not a huge issue but worthy of note.